scriptmodules/helpers.sh

```diff
@@ -345,9 +345,12 @@ function ensureRootdirExists() {
 
     # make sure we have inifuncs.sh in place and that it is up to date
     mkdir -p "$rootdir/lib"
-    if [[ ! -f "$rootdir/lib/inifuncs.sh" || "$rootdir/lib/inifuncs.sh" -ot "$scriptdir/scriptmodules/inifuncs.sh" ]]; then
-        cp --preserve=timestamps "$scriptdir/scriptmodules/inifuncs.sh" "$rootdir/lib/inifuncs.sh"
-    fi
+    declare -a helper_libs=(inifuncs.sh archivefuncs.sh)
```

Comment:
`local helper_libs=(inifuncs.sh archivefuncs.sh)` would suffice. Actually, at some point I intend to do some renaming of vars such as CAPS for global vars and make constants read only, but for now it's fine to leave the declare out.